sample-apps/javascript-sample-app/random-metrics.js

```diff
@@ -36,28 +36,29 @@ let totalHeapSize = 0;
 
 // acquire meter 
 const meter = metricsApi.metrics.getMeter('js-sample-app-meter');
+const instanceID = process.env.INSTANCE_ID
 
 // counter metric
-const timeAliveMetric = meter.createCounter(TIME_ALIVE_METRIC, {
+const timeAliveMetric = meter.createCounter(TIME_ALIVE_METRIC + "_" + instanceID, {
```

Comment:
I think `_instanceID` should only be added if `instance id` is present in the environment. Right now you add the `_` even if it is not present.

Reply:
Since the `testingId` is appended onto the end of the metrics in the expected data template already [here](https://github.com/aws-observability/aws-otel-test-framework/blob/53b4088445e770aba2fc5121285627bd5563e02e/validator/src/main/resources/expected-data-template/standard/standardExpectedMetric.mustache#L2), the sample apps must add the `instanceID` (which is equal to the `testingId`) to the end as well.  This `instance id` environment variable will always be provided because it is provided through the environment in the workflow.

Reply:
You are making the assumption that this sample app will always be ran in the testing environment. Why would I set the `instance id` environment variable if I am just playing around with the sample app on my local machine?

Reply:
Ah ok in that case, it would be cleaner to not add the `_` if the sample app is running locally on the machine.  I'll make that change now.